test/verify/check-users

```diff
@@ -758,6 +758,20 @@ class TestAccounts(MachineCase):
         b.click(".pf-c-select__menu li:contains(testgroup0) button")
         b.wait_not_in_text(".pf-c-label-group__list", "testgroup0")
 
+        # Clicking on the undo button will add the removed group back
+        b.click("#group-undo-btn")
+        b.wait_in_text(".pf-c-label-group__list", "testgroup0")
+        b.click("#account-groups")
+        b.click(".pf-c-select__menu li:contains(testgroup0) button")
+        b.wait_not_in_text(".pf-c-label-group__list", "testgroup0")
+
+        # Clicking on the undo button will remove the added group back
+        b.click("#account-groups")
+        b.click(".pf-c-select__menu li:contains(testgroup0) button")
+        b.wait_in_text(".pf-c-label-group__list", "testgroup0")
+        b.click("#group-undo-btn")
+        b.wait_not_in_text(".pf-c-label-group__list", "testgroup0")
+
```

Comment:
I think the test should also check that it actually is (not) in the groups, not just in the UI

Reply:
Done!